packages/react-components/src/trace-explorer/trace-explorer-time-range-data-widget.tsx

```diff
@@ -200,7 +176,8 @@ export class ReactTimeRangeDataWidget extends React.Component<
         v1 = BigInt(v1);
         v2 = BigInt(v2);
 
-        const offset = this.state.activeData?.absoluteRange?.start || BigInt(0);
+        let offset = this.state.activeData?.absoluteRange?.start || BigInt(0);
```

Comment:
can this be const offset = BigInt(this.state.activeData?.absoluteRange?.start || 0);?